interfaces/automated-testing/geo-awareness/geo-awareness.yaml

```diff
@@ -110,6 +110,220 @@ components:
           example: |-
             Unable to download the dataset https://caa.example.com/geozones.json. Connection refused.
 
+    #########################################################################
+    #################### Start of ED-269 definitions    #####################
+    #########################################################################
+    verticalReferenceType:
+      type: string
+      enum:
+        - AGL
+        - AMSL
+
+    polygonType:
+      type: object
+      description: See ED-269 standard for details.
+      required:
+        - type
+        - coordinates
+      properties:
+        type:
+          type: string
+          enum:
+            - Polygon
+        coordinates:
+          type: array
+          items:
+            type: array
+            minItems: 4
+            items:
+              type: array
+              minItems: 2
+              maxItems: 2
+              items:
+                type: number
+
+    circleType:
+      type: object
+      description: See ED-269 standard for details.
+      required:
+        - type
+        - center
+        - radius
+      properties:
+        type:
+          type: string
+          enum:
+            - Circle
+        center:
+          type: array
+          minItems: 2
+          maxItems: 2
+          items:
+            type: number
+        radius:
+          type: number
+          exclusiveMinimum: 0
```

Comment:
@BenjaminPelletier This field is invalid. How would recommend to handle it ? Should we leave it as is since it is defined in the standard like this or adapt it to reflect the intention of the standard?

Reply:
Addressed by 54b818b